src/data/faq_de.json

```diff
@@ -818,7 +818,7 @@
                             "Nein. Die nun geschaffene Mitnutzung der luca-QR-Codes durch die Corona-Warn-App, die für Eventregistrierungen und Check-ins eingesetzt werden kann, bringt keine weiteren Änderungen für die Nutzenden mit sich.",
                             "CWA-Nutzende können weiterhin ohne Eingabe von Kontaktdaten am CWA-System teilnehmen, während die Kontaktdaten von luca-Nutzenden wie gewohnt im Fall einer Kontaktnachverfolgung ausschließlich vom zuständigen Gesundheitsamt entschlüsselt werden können.",
                             "<b>3. Was passiert beim Scan der QR-Codes durch die Corona-Warn-App?</b>",
-                            "Der QR-Code zur Eventregistrierung bzw. zum Check-in kann mit der Corona-Warn-App (CWA) oder mit luca eingelesen werden.",
+                            "Die von der luca-App generierten QR-Codes für den Check-in-Prozess können sowohl von der Corona-Warn-App (CWA) als auch von luca eingelesen werden.",

```

Comment:
I'm not sure why we have this sentence there in the first place? It is not an answer to the question.